ssn/rdf/ontology/core/sosa-sampling.ttl

```diff
@@ -202,6 +202,30 @@ sosa:Sampling
   skos:example "Taking a diamond-drill core from a rock outcrop."@en ;
   rdfs:isDefinedBy sosa-sam: .
 
+sosa:SamplingCollection 
+  a owl:Class ;
+  rdfs:label "Sampling Collection"@en ;
+  skos:definition """
+  Collection of one or more samplings or sampling collections, i.e. collections may be nested. 
+
+  The following Sampling properties may appear in an Sampling Collection: 
```

Comment:
a, not an.

Reply:
Agreed, recurring problem, also identified above